packages/kit/src/components/Hardware/Hardware.tsx

```
@@ -307,6 +307,103 @@ function MacBluetoothIllustrationViews({
   );
 }
 
+
+function WindowsBluetoothIllustrationViews({
```

ESLint

[error] Delete `⏎`

(prettier/prettier)